README.rst

```diff
@@ -26,8 +26,11 @@ Doom 2                  Not Tested
 .. _Doom (1993): https://en.wikipedia.org/wiki/Doom_(1993_video_game)
 .. _Zephyr RTOS: https://zephyrproject.org/
 
+.. _Visual Studio Code: https://code.visualstudio.com/download
+.. _nRF Connect for VS Code: https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-VS-Code/Download#infotabs
+
 Table of Contents
-^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------------
 
 - `Status`_
 - `Getting Started`_
```

Comment:
I know that this is not in the scope of this change, but "Hardware" section of Table of Contents does not point anywhere and the link is not displayed properly as a result. I suggest just removing this from TOC until we actually have this section.

Reply:
Good catch, fixed.